app/src/ereferrals/java/org/eyeseetea/malariacare/data/sync/exporter/ConvertToWSVisitor.java

```diff
@@ -44,10 +44,12 @@
 
     private SurveyContainerWSObject mSurveyContainerWSObject;
     private String language;
+    private Context mContext;
 
-    public ConvertToWSVisitor() {
+    public ConvertToWSVisitor(Context context) {
```

Comment:
@manuelplazaspalacio instead of making the visitor depend on the context, could you please just add the metadata update date between brackets just after the config file version number? Something like "Config Version: 4 (06/10/2018)". This way you don't need to pass the context through all the classes just to translate one string.